app/assets/stylesheets/global.sass

```diff
@@ -1785,8 +1831,12 @@ form
   .alias-options
     display: none
   .profile-description, .sub-profile-description
-    margin: 5px 5px 15px 5px
+    font-family: "Merriweather Sans"
```

Comment:
`stylesheets/custom_variables.scss` has a `$font-family-sans-serif` where `Merriweather Sans` and a fallback is set, I think it would be cool to use it here